randovania/games/dread/json_data/Dairon.txt

```diff
@@ -1085,10 +1120,12 @@ Extra - asset_id: collision_camera_012
           Any of the following:
```

Comment:
How about adding this path: "Ledge before Slide Jump" -> "Door to Early Grapple Access"?

This would provide a non-lava method of going left without grapple beam. Should be doable with flash shift / spin boost at least.

Reply:
done, don't know why I didn't do this myself. I even thought about it, just dont know why i didnt do it